### DISCUSSION TITLE
Session example

### DISCUSSION THREAD

**Discussion by jesus-beltran** [HashLoad/horse#387] [Q&A]
Please, anyone could provide an example on how to implement sessions on Delphi version of Horse? I just need to figure out the sintax to store/fetch data from the session.

Thanks!!!

**Comment by viniciussanchez** [comment]
Req.Session(<<here!>>), example: Req.Session(TJSONObject.Create.AddPair('id', 1)); to define session...
Req.Session<Type>, example: Req.Session<TJSONObject> to get session on request.

**Comment by ronaldobim** [comment]
You can use a redis for store data session user.

Em seg., 6 de mai. de 2024 19:26, jesus-beltran ***@***.***>
escreveu:
